### PR TITLE
adding role restriction info

### PR DESCRIPTION
**Pull request by crgrieve**

### CODE REVIEW

**Comment by dampee** [comment]
Hi @crgrieve 
I believe you.  Let's merge this.  :-)

**Comment by dampee** [comment]
Hi @crgrieve 
I believe you.  Let's merge this.  :-)

File: Extending/Content-Apps/index-v8.md [hunk 1]
```diff
@@ -112,6 +112,8 @@ In a similar way you can limit your content app according to user roles (groups)
         "+role/admin"  // show for 'admin' user group
     ]
 
+If a role restriction is given in the manifest, it overrides any other restrictions on based on type.
```
Comment (jmayntzhusen) [inline Extending/Content-Apps/index-v8.md]:
```suggestion
If a role restriction is given in the manifest, it overrides any other restrictions based on type.
```

Comment (dampee) [reply Extending/Content-Apps/index-v8.md]: @crgrieve any chance you can accept the suggestion?

Comment (crgrieve) [reply Extending/Content-Apps/index-v8.md]: Done!

**Comment by jmayntzhusen** [comment]
Thanks for adding to the documentation 🎉

**Pull request merged**